cmd/capslock-git-diff/main.go

```diff
@@ -0,0 +1,448 @@
+// Copyright 2024 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// capslock-git-diff lists package capabilities that were added between
+// two revisions of a git repository.
+//
+// Usage example:
+//
+//	capslock-git-diff main mybranch somepath/...
+//
+// This requires Capslock to be installed:
+//
+//	go install github.com/google/capslock/cmd/capslock@latest
+//
+// To compare against the current state of the repository, specify "." as a
+// revision:
+//
+//	capslock-git-diff main . somepath/...
+//
+// If only two arguments are supplied, all packages under the current directory
+// are used.
+//
+// If the environment variable CAPSLOCKTOOLSTMPDIR is set and non-empty, it
+// specifies the directory where temporary files are created.  Otherwise the
+// system temporary directory is used.
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"slices"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	cpb "github.com/google/capslock/proto"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+var (
+	verbose          = flag.Bool("v", false, "enable verbose logging")
+	granularity      = flag.String("granularity", "intermediate", "the granularity to use for comparisons")
+	flagCapabilities = flag.String("capabilities", "-UNANALYZED,-REFLECT,-UNSAFE_POINTER", "if non-empty, a comma-separated list of capabilities to pass to capslock")
```

Comment:
Should the default input only be "-UNANALYZED"

Reply:
Sounds reasonable, done.